### PR TITLE
Merge Docs and Release Notes subteams

### PR DESCRIPTION
**Pull request by katcosgrove**
This PR merges the remainder of the Release Notes responsibilities with the Docs team.

#### What type of PR is this:

/kind documentation

#### What this PR does / why we need it:

Release Notes has very little to do in the early parts of the release and less than before in the middle to late parts of the release due to the moving of Release Highlights to the Comms team. Improvements over time made to the Docs team processes means they now have time to take on these responsibilities.

#### Which issue(s) this PR fixes:

None

cc: @kubernetes/sig-release-leads @gracenng @satyampsoni @npolshakova @drewhagen

### CODE REVIEW

File: release-team/role-handbooks/docs/README.md
Comment (npolshakova) [inline release-team/role-handbooks/docs/README.md:138]: Do we want to include a reference to the old release notes tool here? I don't think it's used by the release notes team anymore. If we want to include historical context, maybe we should link to Marko's talk on the release tooling: https://www.youtube.com/watch?v=ILpvdVDcSTU

File: release-team/role-handbooks/docs/README.md
Comment (npolshakova) [inline release-team/role-handbooks/docs/README.md:139]: Everything since https://github.com/kubernetes/release/issues/1087 went in is handled automatically for the release notes website update. Once the json is available in the bucket, the website should just update. There aren't any manual steps the release team needs to do to update the site. 

Maybe we can change the note to reflect there's no manual steps required?

```suggestion
- [Release notes website](https://relnotes.k8s.io) *(note: release notes website is automatically updated)*
```

File: release-team/role-handbooks/docs/README.md
Comment (npolshakova) [inline release-team/role-handbooks/docs/README.md:177]: @Vyom-Yadav is actively working on adding this! 😄 https://github.com/kubernetes-sigs/release-actions/pull/101 Once it goes in, we should remove this section and add a section on reading the errors reported from the GitHub workflow.

File: release-team/role-handbooks/docs/editing-flow.md
Comment (npolshakova) [inline release-team/role-handbooks/docs/editing-flow.md:56]: Do we want to mention the `--fork` flag? https://github.com/kubernetes/release/blob/master/docs/krel/release-notes.md#examples

Comment (npolshakova) [reply release-team/role-handbooks/docs/editing-flow.md:56]: @satyampsoni

File: release-team/role-handbooks/docs/README.md
Comment (rayandas) [inline release-team/role-handbooks/docs/README.md:122]: We should also mention that`jq` is a hard requirement for running the `krel` command. This is not documented anywhere, and my fellow shadows encountered issues simply because they didn’t have `jq` installed.
@satyampsoni @npolshakova

**Review by rayandas — CHANGES_REQUESTED** [review]

File: release-team/role-handbooks/docs/README.md
Comment (gracenng) [inline release-team/role-handbooks/docs/README.md:25]:
```suggestion
If there are potential fixes to the issues indicated and team members are keen, fixes and automation of the process are encouraged but not required.
```

File: release-team/role-handbooks/docs/README.md
Comment (gracenng) [inline release-team/role-handbooks/docs/README.md:111]:
```suggestion
- Familiarity with Git and command line tools.
- Project management experience is helpful but not required.
```

File: release-team/role-handbooks/docs/README.md
Comment (gracenng) [inline release-team/role-handbooks/docs/README.md:142]: I wasn't aware that contribution for LWKD is part of this role. Is this still the case?

File: release-team/role-handbooks/docs/editing-flow.md
Comment (satyampsoni) [reply release-team/role-handbooks/docs/editing-flow.md:56]: Yes @npolshakova

File: release-team/role-handbooks/docs/README.md
Comment (katcosgrove) [reply release-team/role-handbooks/docs/README.md:139]: If it's no longer used at all, even in edge cases, it doesn't need to be included and I'll remove it. Thanks!

File: release-team/role-handbooks/docs/editing-flow.md
Comment (katcosgrove) [reply release-team/role-handbooks/docs/editing-flow.md:56]: Would one of you like to add a suggestion via review?

**Comment by katcosgrove** [comment]
/retitle Merge Docs and Release Notes subteams
/hold

File: release-team/role-handbooks/docs/README.md
Comment (npolshakova) [inline release-team/role-handbooks/docs/README.md:75]: Should we include going over krel as part of the onboarding session? 

Also @satyampsoni was creating a assignment sheet useful or is this something we should drop? https://github.com/kubernetes/sig-release/tree/master/release-team/role-handbooks/release-notes#week-0

File: release-team/role-handbooks/docs/README.md
Comment (npolshakova) [inline release-team/role-handbooks/docs/README.md:89]: Is this missing `alpha` intentionally? 
```suggestion
The `release-notes` subcommand of `krel` must continue to be run on the release branch (for `alpha`, `beta` and `rc` releases) in order to pull in any outstanding PRs that are merged between the beginning of code freeze and the release.
```

File: release-team/role-handbooks/docs/README.md
Comment (npolshakova) [reply release-team/role-handbooks/docs/README.md:123]: @rayandas did you ever track down what the error was?

File: release-team/role-handbooks/docs/README.md
Comment (npolshakova) [reply release-team/role-handbooks/docs/README.md:139]: Yep, it's not used by shadows creating the release notes so I think we can remove it.

File: release-team/role-handbooks/docs/README.md
Comment (npolshakova) [inline release-team/role-handbooks/docs/README.md:71]: Can we add a bullet point to highlight the final release notes PR creation and reaching out to the sigs (via slack and email) to ask them to review the final draft?
 ```suggestion
- Up to 1 hour weekly to attend [SIG Docs meetings](https://github.com/kubernetes/community/tree/master/sig-docs#meetings) for status reports
- Create [known issues issue](https://github.com/kubernetes/sig-release/blob/master/release-team/role-handbooks/release-notes/known-issues-bucket.md) in kubernetes/kubernetes to capture known issues for the release
- Send [an email to SIG-leads](https://github.com/kubernetes/sig-release/blob/master/release-team/role-handbooks/release-notes/sig-leads-email.md) to ensure major changes for their SIGs are accurately reflected in the release notes
- Send a slack message to the sig channels to ensure major changes for the SIGs are accurately reflected in the release notes 
```

File: release-team/role-handbooks/docs/README.md
Comment (rayandas) [reply release-team/role-handbooks/docs/README.md:123]: @npolshakova I didn’t. But I will try to regenerate it today. And let you know.

File: release-team/role-handbooks/docs/README.md
Comment (katcosgrove) [reply release-team/role-handbooks/docs/README.md:178]: @Vyom-Yadav Should this be left in for this release or can it be removed?

Comment (npolshakova) [reply release-team/role-handbooks/docs/README.md:181]: Let's leave it for now. We can remove it once the workflow merges in sig-release

File: release-team/role-handbooks/docs/editing-flow.md
Comment (npolshakova) [inline release-team/role-handbooks/docs/editing-flow.md:76]:
```suggestion

The `--fork` flag should be set to your GitHub user (the organization that owns your fork of k/sig-release). This is required to automatically create the release notes branch on your fork of sig-release.

```

File: release-team/role-handbooks/docs/editing-flow.md
Comment (npolshakova) [reply release-team/role-handbooks/docs/editing-flow.md:56]: Added:
- https://github.com/kubernetes/sig-release/pull/2665/files#r1889101741
- https://github.com/kubernetes/sig-release/pull/2665/files#r1889103626

**Comment by katcosgrove** [comment]
/unhold

@gracenng, the stage is yours

**Review by npolshakova — APPROVED** [review]
Can make additional edits after these initial changes go in as needed.

**Comment by gracenng** [comment]
Merging this for shadow applications and we'll continue working on it after

/approve
/lgtm

File: release-team/role-handbooks/docs/README.md
Comment (rayandas) [reply release-team/role-handbooks/docs/README.md:126]: I couldn’t regenerate it on my system. I will keep an eye on errors faced by new contributors around this and then report it with details.

**Pull request merged**
